db/controllor/controller.js

```diff
@@ -11,7 +11,10 @@ const getTopics = (request, response, next) => {
 };
 
 const getArticles = (request, response, next) => {
-    selectAllArticles()
+    const { sort_by } = request.query;
+    const { order } = request.query;
+    const { topic } = request.query;
```

Comment:
just a handy tip that these can be combined within one of the objects as their all being deconstructed from the same request.query